mkdocs.yml

```diff
@@ -36,10 +36,38 @@ markdown_extensions:
 plugins:
   - search
   - mkdocs-video
-  - redirects:
-      redirect_maps:
-        'common-coreboot-docs/dasharo_tools_suite.md': 'dasharo-tools-suite/overview.md'
-        'variants/clevo_nv41/ec_update.md': 'dasharo-tools-suite/documentation.md'
+  - exclude:
```

Comment:
@Asiderr why we're excluding below sites. Is there an option to add comment in the mkdocs.yml that explains this parameter? https://stackoverflow.com/questions/2276572/how-do-you-do-block-comments-in-yaml

Reply:
I added comment https://github.com/Dasharo/docs/pull/582/commits/fd5d9c0db700928b5915dabfd0f795e1e651f968